src/ontology/mondo-edit.obo

```diff
@@ -431419,20 +431419,32 @@ replaced_by: MONDO:0005570
 [Term]
 id: MONDO:0020007
 name: absence of the pulmonary artery
-def: "Unilateral absence of the pulmonary artery (UAPA) is a rare congenital great vessels anomaly that commonly presents by dyspnea, frequent respiratory infections, hemoptysis and high-altitude pulmonary edema. UAPA is often associated with congenital heart malformation (CHM). In the absence of associated cardiac malformation (isolated UAPA; IUAPA), the condition may be asymptomatic until adult age." [Orphanet:980]
+def: "A rare vascular anomaly characterized by congenital absence of the right or left pulmonary artery, usually ending within 2 cm of its expected origin from the pulmonary trunk. Patients often also have other cardiovascular abnormalities and respective symptoms and are then typically diagnosed in infancy or childhood, while isolated cases generally present with a mild clinical course and may go undiagnosed until adulthood. Presenting clinical features in isolated cases include hemoptysis, exertional dyspnea, and recurrent respiratory infections. The condition is typically accompanied by marked changes of lung tissue and may, if unrecognized, result in massive hemoptysis and pulmonary hypertension." [Orphanet:980]
 subset: gard_rare {source="GARD:18712"}
 subset: nord_rare {source="MONDO:NORD"}
 subset: ordo_morphological_anomaly {source="Orphanet:980"}
 subset: orphanet_rare {source="Orphanet:980"}
 subset: rare
+synonym: "agenesis of pulmonary artery" EXACT []
+synonym: "aplasia of pulmonary artery" EXACT [Orphanet:980]
+synonym: "congenital absence of pulmonary artery" RELATED []
+synonym: "pulmonary artery absent" EXACT []
+synonym: "pulmonary artery agenesis" EXACT [MONDO:0043187]
```

Comment:
@nicolevasilevsky These synonyms don't have a x-ref because they didn't have an x-ref on the other term). 
Maybe we could add your orcid?